<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <layout class="QVBoxLayout" name="outerLayout">
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>32</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>32</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>SWC DBMS Client</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string>font-size: 20px; font-weight: bold; color: #2979ff; border: none; background: transparent;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="subtitleLabel">
     <property name="text">
      <string>Sign in to your account</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string>color: #888888; font-size: 13px; border: none; background: transparent;</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="topSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="userLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="styleSheet">
        <string>border: none; background: transparent;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="userNameEditor">
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="passwordLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="styleSheet">
        <string>border: none; background: transparent;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordEditor">
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="midSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>12</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="loginBtn">
     <property name="text">
      <string>Sign In</string>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="registerLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>No account?</string>
       </property>
       <property name="styleSheet">
        <string>color: #888888; border: none; background: transparent;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="registerBtn">
       <property name="text">
        <string>Register Now</string>
       </property>
       <property name="styleSheet">
        <string>border: none; background: transparent; color: #2979ff; font-weight: bold; text-align: left; padding: 0;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="registerSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
